Automattic/wp-calypso
# Plugin Details: adds sidebar usps.

#### Changes proposed in this Pull Request

* added static sidebar USPs for paid plugins.
* removed unused svg
* added "accordion" as search keyword for `FoldableCard` component.
* demo & documentation will not yet appear as they don't exist in the response which will be added here https://github.com/Automattic/wp-calypso/issues/58701. But they have been implemented 

#### Testing instructions



|Desktop | Mobile|
|-------|------|
|<img width="1083" alt="SS 2021-12-22 at 12 23 52" src="https://user-images.githubusercontent.com/12430020/147077798-2dfa3a8a-08ee-45b0-9a1d-ec87a78158cd.png">|![SS 2021-12-22 at 12 24 35](https://user-images.githubusercontent.com/12430020/147077884-f9e60074-d2b0-422a-b14f-2e2c9249a6c7.gif)|

* visit a paid plugin
* check that USPs appear (demo & documentation will not yet appear as they don't exist in the response) 
* visit a free plugin
* check that plugin details appear



Fixes #57758
